src/Symfony/Component/AssetMapper/Factory/MappedAssetFactory.php

```diff
@@ -25,6 +26,7 @@ class MappedAssetFactory implements MappedAssetFactoryInterface
 {
     private const PREDIGESTED_REGEX = '/-([0-9a-zA-Z]{7,128}\.digested)/';
     private const PUBLIC_DIGEST_LENGTH = 7;
+    private const INTEGRITY_HASH_ALGORITHMS = ['sha256', 'sha384', 'sha512'];
```

Comment:
These names are actually both the integrity prefix and the algorithm. For now they all match so as the list is private I guess we'll update it to a map if needed.